jerry-core/vm/vm.c

```diff
@@ -1725,12 +1725,15 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           {
             left_value = ECMA_VALUE_UNDEFINED;
             right_value = ECMA_VALUE_UNDEFINED;
+          }
```

Comment:
The indentation is still off by two spaces from line 1725 to 1738.

Reply:
Am I missing something? I think the indentation is correct. All the 3 if statements must be at the same level with the

```c
  stack_top_p += 2;
  left_value = result;
  right_value = ECMA_VALUE_UNDEFINED;
  /* FALLTHRU */
```

part.

Reply:
The glitch is between lines 1724 and 1725.

Reply:
I see now! That's a github glitch, that I've already met several times before. See the related [part](https://github.com/rerobika/jerryscript/blob/fix_issue_2654/jerry-core/vm/vm.c#L1724) from my branch where the indentation is correct! :)

Reply:
Indeed. ("View file" also reveals that indentation is correct.)